.travis.yml

```diff
@@ -0,0 +1,36 @@
+# Only building master means that we don't run two builds for each pull request.
+branches:
+  only:
+  - master
+
+language: node_js
+
+node_js:
+- node
+- lts/fermium
+- lts/erbium
+
+os:
+- linux
+- osx
+- windows
+
+env:
+  global:
+  - PROTOBUF_VERSION=3.13.0
+  - PATH="$HOME/protoc/bin:$PATH"
+
+cache: npm
+
+install:
+- if [[ "$TRAVIS_OS_NAME" = windows ]]; then
```

Comment:
Can you add a comment here explaining why we can't use the protoc npm package?

Reply:
No good reason, it makes sense to do this through npm.